helper.go

```diff
@@ -328,23 +332,8 @@ func getCommand(shellPrompt string) {
 		userInput = strings.TrimSpace(userInput)
 		if userInput == "y" || userInput == "" {
 			var cmd *exec.Cmd
-			if runtime.GOOS == "windows" {
-				shellName := "cmd"
-
-				if len(os.Getenv("PSModulePath")) > 0 {
-					shellName = "powershell"
-				}
-				if shellName == "cmd" {
-					cmd = exec.Command("cmd", "/C", fullLine)
-
-				} else {
-					cmd = exec.Command("powershell", fullLine)
-				}
-
-			} else {
-				cmd = exec.Command("bash", "-c", fullLine)
-
-			}
+			// Directly use the shellName variable set by setShellAndOSVars()
+			cmd = exec.Command(shellName, "-c", fullLine)
```

Comment:
Everything looks good, just there is no code for windows

Reply:
Oh, I thought I pushed the changes. Let me push my local repo once more. Sorry